Arrangement_on_surface_2/include/CGAL/Arr_geometry_traits/Bezier_cache.h

```diff
@@ -170,6 +170,7 @@ template <class Nt_traits_> class _Bezier_cache
                                           tangency parameters. */
   Intersect_map     intersect_map;    /*! Maps curve pairs to their
                                           intersection parameter pairs. */
+  std::pair<Polynomial, Polynomial>     resultant_pair;
```

Comment:
The resultant polynomial of the two bezier curves. Need to store them to do the boundary check.